Marlin/src/inc/Conditionals_LCD.h

```diff
@@ -651,6 +651,9 @@
 
 #if ENABLED(DWIN_CREALITY_LCD)
   #define SERIAL_CATCHALL 0
+  #ifndef LCD_SERIAL_PORT
+    #define LCD_SERIAL_PORT 3 // Creality 4.x board
```

Comment:
Should this not be `1`

Reply:
The Ender-3 V2 configuration sets `SERIAL_PORT_2` to 3, so that is the default here.

Reply:
Gotcha.  Perhaps this is another board, but below it was using `MYSERIAL1`.  So my expectation would be that `LCD_SERIAL_PORT 3` would translate to `MYSERIAL3` no?

Apologies if I'm being a bit annoying here, just double checking :)

Reply:
Serial ports are typically defined in the HAL like so…
```cpp
#define MYSERIAL1 MSERIAL(SERIAL_PORT_2)
```
So `MYSERIAL0` always corresponds to ` SERIAL_PORT`; `MYSERIAL1` always corresponds to `SERIAL_PORT_2`; etc.